src/main/java/edu/cornell/kfs/pdp/document/CuPayeeACHAccountMaintainableImpl.java

```diff
@@ -64,12 +69,25 @@ public boolean answerSplitNodeQuestion(String nodeName) {
         if (StringUtils.equalsIgnoreCase(CUPdpConstants.PAYEE_ACH_ACCOUNT_REQUIRES_PDP_APPROVAL_NODE, nodeName)) {
             return payeeACHAccountMaintenanceRequiresPdpApproval();
         }
-        
         return super.answerSplitNodeQuestion(nodeName);
     }
 
     protected boolean payeeACHAccountMaintenanceRequiresPdpApproval() {
+        try {
+            DocumentService documentService = SpringContext.getBean(DocumentService.class);
+            FinancialSystemMaintenanceDocument document = (FinancialSystemMaintenanceDocument) documentService.getByDocumentHeaderId(this.getDocumentNumber());
+            if (isDocumentInitiatorSystemUser(document)) {
```

Comment:
The whole block for the is-sys-user check should be moved below the is-edit-action check. As per the requirements from the user story, PAAT edits should *ALWAYS* route for approval. You should adjust this method so that it short-circuits if the is-edit-action check returns true, and only proceed to the is-sys-user check if the is-edit-action check returns false.